docs/dev/adding_instruments.rst

```diff
@@ -693,3 +693,29 @@ Another use case involves maintaining compatibility between instruments with com
 
 In the above example, :code:`MultimeterA` and :code:`MultimeterB` use a different command to read the voltage, but the rest of the behaviour is identical.
 :code:`MultimeterB` can be defined subclassing :code:`MultimeterA` and just implementing the difference.
+
+
+Naming derived instruments
+**************************
+
+When deriving an instrument from an existing implementation, you might want to give separate names to any derived instrument. Two possible schemes for this are the following: 
+
+.. code-block:: python
+    class BaseClass:
```

Comment:
```suggestion
.. code-block:: python

    class BaseClass:
```
This needs at least this to work -- in RST there are blank lines between directives/their options and the content.